dvc/output/base.py

```diff
@@ -67,7 +67,7 @@ def __init__(
         self.repo = stage.repo
         self.url = path
         self.info = info
-        self.remote = remote or self.REMOTE(self.repo, {})
+        self.remote = remote or self.REMOTE(self.repo, {}, "output")
```

Comment:
does not seem that `output` is the right name here. Should be it just cache in this case?